miner/worker.go

```diff
@@ -92,669 +54,190 @@ type environment struct {
 	txs      []*types.Transaction
 	receipts []*types.Receipt
 	sidecars []*types.BlobTxSidecar
+	logs     [][]*types.Log
```

Comment:
It seems like this is only used to fill `newPayloadResult.logs`, which is also not consumed by anything? Is there an intended use for the logs here?